substra/cli/interface.py

```diff
@@ -207,6 +207,31 @@ def load_data_samples_keys(data_samples, option="--data-samples-path"):
         raise click.BadParameter('File must contain a "keys" attribute.', param_hint=f'"{option}"')
 
 
+def _error_message(fn, errors):
+    action = fn.__name__.replace('_', ' ')
+
+    def _error_lines(errors_):
+        lines_ = []
+        for field, field_errors in errors_.items():
+            for field_error in field_errors:
+                lines_.append(f"- {field}: {field_error}")
+        return lines_
+
+    lines = []
+    if isinstance(errors, dict):
+        lines += _error_lines(errors)
+    elif isinstance(errors, list):
+        for error in errors:
+            lines += _error_lines(error)
+    else:
+        lines.append(f"- {errors}")
```

Comment:
minor: I would handled the add of the "- " in a single place. For instance:

```python
    lines.append(str(errors))
pluralized_errors = ...
return "Could not..." + "\n- ".join(lines)
```

Reply:
done.